%YAML:1.0
---
CameraMat: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 428.866631416691, 0., 328.911940731370, 0., 428.887235861938, 239.638162195323, 0., 0., 1. ]
DistCoeff: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 0.173737395080563, -0.295181584840770, -0.00501817437570227, -0.00209038560993722, 0.209661580427388 ]
ImageSize: [ 640, 480 ]
MeanReprojError: 0.075457038553250
CameraExtrinsicMat: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 0.01139046, -0.99992442, 0.00386271, 0.00764093, -0.00467308, -0.00391612, -0.99997979, 0.07553916, 0.99992228, 0.01137163, -0.00471738, 0.02120813, 0.00000000, 0.00000000, 0.00000000, 1.00000000 ]
LidarToRefExtrinsicMat: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [-6.756055868811061682e-01, -4.705467037714762824e-01, -5.675763301484961776e-01, 6.551506330541521095e+00, 4.512567152509880941e-01, -8.727152664533444071e-01, 1.863744634872404882e-01, -6.834529831407374356e-01, -5.830304176592545629e-01, -1.302070016130722763e-01, 8.019486696884088328e-01, 1.506730299266465956e+00, 0.000000000000000000e+00, 0.000000000000000000e+00, 0.000000000000000000e+00, 1.000000000000000000e+00 ]